### PR TITLE
0007 check receiving account list

### LINKED ISSUE #7: Check receiving account key list
### Problem

When the receiver signature required flag is on the transaction requires the signatures of the receiving account. Currently, the tool does not check the list of keys on the receiving account to see if there is a key the app could recommend the user to sign with.

### Solution

Scan the receiving account key list for a transfer transaction to see if there are keys that can be recommended for the user to sign with. 

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by DSMatusevich**
**Description**:
The application now scans the receiving account key list for a transfer transaction to see if any of the accounts have the `receiverSigRequired` flag. If any account does, it is added to the set of signing accounts.

**Related issue(s)**:

Closes #7 

**Checklist**

- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by SimiHunjan — APPROVED** [review]

File: tools-core/src/main/java/com/hedera/hashgraph/client/core/transactions/ToolTransferTransaction.java [hunk 1]
```diff
@@ -147,10 +152,24 @@ public boolean checkInput(JsonObject input) throws HederaClientRuntimeException
 
 	@Override
 	public Set<AccountId> getSigningAccounts() {
+		Map<AccountId, AccountInfo> infos = new HashMap<>();
+		try {
+			infos = loadAccountInfos();
+		} catch (HederaClientException | InvalidProtocolBufferException e) {
+			logger.error(e.getMessage());
```
Comment (tinker-michaelj) [inline tools-core/src/main/java/com/hedera/hashgraph/client/core/transactions/ToolTransferTransaction.java:159]: It looks like if we hit this `catch`, we can continue---**but** may omit some necessary signatures for involved accounts with `receiverSigRequired=true`.

So perhaps a `WARN`-level log would be more appropriate, with a message that describes the user impact of the event. E.g.,
```suggestion
			logger.warn("Unable to load account information, some required receiver signatures may be omitted", e);
```

**Review by tinker-michaelj — APPROVED** [review]
LGTM! Made one suggestion on logging usage.

File: tools-core/src/main/java/com/hedera/hashgraph/client/core/transactions/ToolTransferTransaction.java (hunk 1 above)
Comment (DSMatusevich) [reply tools-core/src/main/java/com/hedera/hashgraph/client/core/transactions/ToolTransferTransaction.java:159]: Will make the change.

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**